script/1_0_2current_en_script/Arcueid/Day 2/02_00_ARC02_1_1.txt

```diff
@@ -17,13 +17,13 @@ The teacher who walks into the classroom is a person that I had never seen befor
 <Page6>
 "Huh? What's up with this atmosphere?
 Everyone froze up all of a sudden...
-What am I, a 'Basilisk' woman?# Oh wait, that metaphor is a bit obscure, isn't it? Erm, in Japanese terms, maybe something like the girl from 'The Ring'? But Sensei's name isn't Sadako, you know."
+What am I, a basilisk woman?# Oh wait, that metaphor is a bit obscure, isn't it? Erm, in Japanese terms, maybe something like the girl from 'The Ring'? But Sensei's name isn't Sadako, you know." //I don't think "basilisk" needs quotation marks or capitalization. 
 <Page7>
 'Isn't that metaphor a bit old?', I thought in my head while raising an eyebrow.
 Meanwhile, the mysterious woman stands at the podium without a hint of concern and clears her throat deliberately.
 <Page8>
-"Nice to meet you. I'm Noel Aizome, the new substitute for Toyama-sensei who has been transferred.
-We'll be hanging out for only a month, but I'm looking forward to working with you all."
+"Nice to meet you. I'm Noel Aizome, the new substitute for the recently transferred Toyama-sensei. //Trying to rewrite this without the overly long clause at the end.
```

Comment:
Maybe reorganize the sentence a bit to smooth it out -
>はじめまして。戸山先生が転任なされたので、その代行としてやってきた新任の<愛染|あいぞめ>ノエルです。
> Nice to meet you, my name is Noel Aizome. Since Toyama-sensei has transferred somewhere else, I'll standing in for her temporarily.

Unrelated, is her name supposed to be flopped? We refer to Tohno Shiki in Surname-Firstname order, shouldn't Aizome Noel be consistent with that?

Reply:
Well, foreigners will sometimes keep their first and last name in the same order when moving to Japan. Presumably she would be going by Aizome-sensei if that wasn't the case.